### PR TITLE
Add geo location to task.

### PR DESCRIPTION
**Pull request by eodolphi**
Add new generic "Place" model that makes it possible to add a place to
a model. It includes address fields and geoposition.

TODO: also use this to store a project's location.

BB-13141 #resolve

### CODE REVIEW

File: bluebottle/tasks/models.py [hunk 1]
```diff
@@ -101,6 +105,15 @@ def owner(self):
     def parent(self):
         return self.project
 
+    @property
+    def place(self):
+        try:
+            return self.places.get()
+        except Place.DoesNotExist:
+            return None
+        import ipdb; ipdb.set_trace()
```
Comment (gannetson) [inline bluebottle/tasks/models.py]: Say wot?

Comment (eodolphi) [reply bluebottle/tasks/models.py]: Haha, die kwam gewoon door de test natuurlijk.

**Comment by eodolphi** [comment]
This still needs indexing of task places.

File: bluebottle/tasks/models.py [hunk 2]
```diff
@@ -101,6 +105,13 @@ def owner(self):
     def parent(self):
         return self.project
 
+    @property
+    def place(self):
+        try:
+            return self.places.get()
```
Comment (gannetson) [inline bluebottle/tasks/models.py]: You could now have multiple places, right?
I think you should also catch `Place.MultipleObjectsReturned` or just do `return self.places.first()` (which is less explicit of course)

Comment (gannetson) [reply bluebottle/tasks/models.py]: Ah ne'er mind you limit it in admin too...

**Pull request merged**